build.gradle

```diff
@@ -11,6 +12,11 @@ repositories {
     mavenCentral()
 }
 
+jacoco {
+    toolVersion = "0.8.12"
```

Comment:
Move this to version.gradle file, where we store all dependency versions

Reply:
Thank you!

Reply:
Moved